sdk/face/azure-ai-vision-face/README.md

```diff
@@ -0,0 +1,484 @@
+        large_person_group_id,
+        bill_person_id,
+        read_file_content(bill_image_file_path),
+        detection_model=FaceDetectionModel.DETECTION_03,
+        user_data="Dad-0001")
+
+    print("Create a Person Clare and add a face to her.")
+    clare_person_id = face_admin_client.create_large_person_group_person(
+        large_person_group_id, name="Clare", user_data="Mom").person_id
+    clare_image_file_path = "./samples/images/Family1-Mom1.jpg"
+    face_admin_client.add_large_person_group_person_face(
+        large_person_group_id,
+        clare_person_id,
+        read_file_content(clare_image_file_path),
+        detection_model=FaceDetectionModel.DETECTION_03,
+        user_data="Mom-0001")
+```
+
+Before doing the identification, we need to train the LargePersonGroup first.
+```python
+    print(f"Start to train the large person group: {large_person_group_id}.")
+    poller = face_admin_client.begin_train_large_person_group(large_person_group_id)
+
+    # Wait for the train operation to be completed.
+    # If the training status isn't succeed, an exception will be thrown from the poller.
+    training_result = poller.result()
+```
+
+When the training operation is completed successfully, we can identify the faces in this LargePersonGroup through
+`FaceClient`.
+```python
+with FaceClient(endpoint=endpoint, credential=AzureKeyCredential(key)) as face_client:
+    # Detect the face from the target image.
+    target_image_file_path = "./samples/images/identification1.jpg"
+    detect_result = face_client.detect(
+        read_file_content(target_image_file_path),
+        detection_model=FaceDetectionModel.DETECTION_03,
+        recognition_model=FaceRecognitionModel.RECOGNITION_04,
+        return_face_id=True)
+    target_face_ids = list(f.face_id for f in detect_result)
+
+    # Identify the faces in the large person group.
+    result = face_client.identify_from_large_person_group(
+        face_ids=target_face_ids, large_person_group_id=large_person_group_id)
+    for idx, r in enumerate(result):
+        print(f"----- Identification result: #{idx+1} -----")
+        print(f"{r.as_dict()}")
+```
+
+Finally, use `FaceAdministrationClient` to remove the large person group if you don't need it anymore.
+```python
+with FaceAdministrationClient(endpoint=endpoint, credential=AzureKeyCredential(key)) as face_admin_client:
+    print(f"Delete the large person group: {large_person_group_id}")
+    face_admin_client.delete_large_person_group(large_person_group_id)
+```
+
+### Liveness detection
+Face Liveness detection can be used to determine if a face in an input video stream is real (live) or fake (spoof).
+The goal of liveness detection is to ensure that the system is interacting with a physically present live person at
+the time of authentication. The whole process of authentication is called a session.
+
+There're two different components in the authentication: a mobile application and an app server/orchestrator.
```

Comment:
Updated the README. For the model naming, I suggest not changing it because our model is called `detection_03`.

Reply:
Add another question here - the [pipeline](https://dev.azure.com/azure-sdk/public/_build/results?buildId=3778471&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=a880e989-7d1a-5c96-a41f-d540b383cc43) found a broken link in README.md which is the package destination: https://pypi.org/project/azure-ai-vision-face/. Because we haven't released this SDK, the link is invalid.

Should I remove this link?

Reply:
You can create an aka.ms link for it and point to a placeholder link. After the release, you can re-point to the actual pypi link.